contrib/gorilla/mux/option.go

```diff
@@ -103,3 +109,36 @@ func WithResourceNamer(namer func(router *Router, req *http.Request) string) Rou
 		cfg.resourceNamer = namer
 	}
 }
+
+// WithHeaderTags enables setting of request header as tags on the span
+func WithHeaderTags() RouterOption {
+	return func(cfg *routerConfig) {
+		cfg.setRequestData = true
+	}
+}
+func getHeaderTags(req *http.Request) ddtrace.StartSpanOption {
```

Comment:
Don't use `get*` in Go please. https://golang.org/doc/effective_go#Getters

Reply:
Extract?

Reply:
I'd suggest `headerTags` or `headerTagsFromRequest`